parser_library/src/workspaces/processor_group.h

```diff
@@ -38,13 +38,22 @@ class processor_group : public diagnosable_impl
 
     void add_library(std::unique_ptr<library> library) { libs_.push_back(std::move(library)); }
 
+    void add_asm_options(std::vector<std::string> asm_options)
+    {
+        asm_options_.insert(make_pair("SYSPARM", asm_options.at(0)));
+        asm_options_.insert(make_pair("PROFILE", asm_options.at(1)));
+    }
+
```

Comment:
I really don't like this interface... It is not obvious that the only correct vector is the one with two elements, no way to tell easily at call site that the first one is sysparm, the other profile...

Reply:
Yes , you are right. Will change it to map to with key values "SYSPARM" and "Profile"